correios/models/data.py

```diff
@@ -544,7 +544,7 @@
     "PO": "Postagem (exceção)",
     "RO": "Expedição de Lista de Registro",
     "TRI": "Triagem",
-    # "CMR": "Evento Desconhecido",  # usage not found in SRO documentation
+    # "CMR": "Evento Desconhecido",  # event type not found in SRO documentation from 2017-05-02
```

Comment:
> What about to change ... from 2017-05-02 for ... since 2017-05-02?

I think both are wrong. What I wanted to say here is that I checked an official correios documentation and this doc was created on 2017-05-02. Which means if a new doc release is made (or if the reader read an only one) the comment may be wrong. And IMO changing the "from" to "since" will make it look like the comment will always be right because since always refers to "today". In 10 years "today" is going to be 2028 and this sentence is going to affirm still hasn't been found in 10 years and we can't be sure of that.  

Maybe changing to `event type not found in the latest SRO documentation (issued on 2017-05-02)` will be better.

Reply:
What do you think?

Reply:
the phrase with "issued on" looks nice to me :+1: 

what about the deal with the date format?

Reply:
> the phrase with "issued on" looks nice to me 👍

indeed. I agree

> what about the deal with the date format?

+1 to change the month to a written format like 'feb', 'may', etc.